File: conference/registration-bc/web/src/main/java/com/microsoft/conference/registration/domain/Orders/Events/OrderRegistrantAssigned.cs
using System;
using ENode.Eventing;

namespace Registration.Orders
{
    public class OrderRegistrantAssigned : DomainEvent<Guid>
    {
        public Guid ConferenceId { get; private set; }
        public Registrant Registrant { get; private set; }

        public OrderRegistrantAssigned() { }
        public OrderRegistrantAssigned(Guid conferenceId, Registrant registrant)
        {
            ConferenceId = conferenceId;
            Registrant = registrant;
        }
    }
}

File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/domain/orders/Events/OrderRegistrantAssigned.java
package com.microsoft.conference.registration.domain.orders.Events;

import com.microsoft.conference.registration.domain.orders.Models.Registrant;
import org.enodeframework.eventing.DomainEvent;

public class OrderRegistrantAssigned extends DomainEvent<String> {
    public String ConferenceId;
    public com.microsoft.conference.registration.domain.orders.Models.Registrant Registrant;

    public OrderRegistrantAssigned() {
    }

    public OrderRegistrantAssigned(String conferenceId, Registrant registrant) {
        ConferenceId = conferenceId;
        Registrant = registrant;
    }
}
